sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/workflow/whenphase/routeinvoker/impl/DefaultRouteInvoker.java

```diff
@@ -14,18 +14,12 @@ public class DefaultRouteInvoker implements RouteInvoker {
   private final CamelContext camelContext;
 
   @Override
-  public Exchange invoke(Exchange exchange) {
-    ExchangeBuilder exchangeBuilder = ExchangeBuilder.anExchange(camelContext);
-    return exchangeBuilder.build();
+  public Exchange invoke(Exchange exchange, Endpoint endpoint) {
+    return SIPExchangeHelper.createEmptyExchange(camelContext);
```

Comment:
Perhaps we should throw an exception if we don't have invoker. Let's discuss

Reply:
DefaultRouteInvoker is removed and exception is thrown instead.